edx_solutions_api_integration/courses/views.py

```diff
@@ -1319,6 +1323,8 @@ def get_queryset(self):
             users = users.prefetch_related(
                 Prefetch('course_groups', queryset=CourseUserGroup.objects.filter(course_id=self.course_key))
             )
+        if User._meta.get_field(order_by_field):
+            users = users.order_by(order_by_field)
```

Comment:
I'm not sure if this will work:
https://docs.djangoproject.com/en/1.8/ref/models/meta/#django.db.models.options.Options.get_field

It raises `FieldDoesNotExist`.

Reply:
Yeah, I think I need to sleep :p